CMakeLists.txt

```diff
@@ -97,6 +97,7 @@ opm_add_test(test_blackoilfluidstate)
 opm_add_test(test_ConditionalStorage)
 opm_add_test(test_eclblackoilfluidsystem CONDITION HAVE_ECL_INPUT)
 opm_add_test(test_eclblackoilpvt CONDITION HAVE_ECL_INPUT)
+opm_add_test(test_eclblackoilpvt_eval CONDITION HAVE_ECL_INPUT)
```

Comment:
BTW: this test already is conditional on the availability of the parser, so nothing needs to be done here...